changes/5172.fixed

```diff
@@ -0,0 +1,2 @@
+Fixed Brand Icon mouseover Background: Fix for mouseover effect on the Brand / Icon (was flashing white background vs being transparent) when in dark mode.
+Fixed Menu Button alignment Behavior: Fix that prevents menu item buttons from wrapping. Previously menu item buttons might stack on top of each other and look misaligned if there were more than one button. 
```

Comment:
#5268 accomplishes button wrapping avoidance by adjusting gap and margins.  This PR was going to ensure there is no `text-wrap` assigned to navbar button menu items, but in further review I think it may no longer need to be added.